userspace/openpilot_dependencies.sh

```diff
@@ -3,6 +3,7 @@
 echo "Installing openpilot dependencies"
 
 # Install necessary libs
+add-apt-repository ppa:deadsnakes/ppa -y 
```

Comment:
why is this needed if we’re using uv?

Reply:
the python3.12-dev package provided by the repo is not needed since uv python also provides .pc files for pkgconfig and the entry can be removed.